destreamer.ts

```diff
@@ -205,12 +198,18 @@ async function downloadVideo(videoUrls: string[], outputDirectory: string, sessi
     }));
 }
 
-// FIXME
-process.on('unhandledRejection', (reason) => {
-    console.error(colors.red('Unhandled error!\nTimeout or fatal error, please check your downloads and try again if necessary.\n'));
-    console.error(colors.red(reason as string));
-    throw new Error('Killing process..\n');
-});
+
+function startup() {

```

Comment:
Let's call that `init()`. Actually i think we already have an init() function introduced few PRs ago, use that if you can.